script/openshift-cluster.mk

```diff
@@ -46,13 +46,30 @@ devel/openshift-kubeapps-project-created: devel/openshift-tiller-project-created
 		oc policy add-role-to-user edit "system:serviceaccount:${TILLER_NAMESPACE}:tiller" && \
 		touch $@
 
-devel/openshift-kubeapps-installed: openshift-install-tiller
+chart/kubeapps/charts/mongodb-%.tgz:
+	helm dep update ./chart/kubeapps
```

Comment:
Otherwise if someone pulls our Kubeapps repo and runs `make openshift-kubeapps` it fails because it's deploying the chart from the current branch, which won't include the chart deps initially.